<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceCfg_ExternalApps</class>
 <widget class="QWidget" name="InterfaceCfg_ExternalApps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-73</y>
        <width>425</width>
        <height>402</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Application 1</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Args</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QCheckBox" name="chckToolBar1">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lePath1"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leArgs1"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnFind1">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/resources/icons/projOpen.png</normaloff>:/resources/icons/projOpen.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Application 2</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Args</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QCheckBox" name="chckToolBar2">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lePath2"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leArgs2"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnFind2">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/resources/icons/projOpen.png</normaloff>:/resources/icons/projOpen.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Application 3</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Args</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QCheckBox" name="chckToolBar3">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lePath3"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leArgs3"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnFind3">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/resources/icons/projOpen.png</normaloff>:/resources/icons/projOpen.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
